tests/pyqtslot.py

```diff
@@ -8,9 +10,84 @@ def func_none(s: str) -> None:
 def func_int(s: str) -> int:
     return 42
 
-@pyqtSlot(str, result='int')
+@pyqtSlot(str, result='int')    # this means int here
 def func_str(s: str) -> str:
     return '42'
 
-func_none("test")
-x = func_int("test")  # type: int
+@pyqtSlot(str, name='toto', revision=33, result=int)
+def func_toto(s: str) -> int:
+    return 33
+
+n = None
+n = func_none("test")
```

Comment:
Is this better than the existing explicit hinting of the variable type?

Reply:
I find it more reliable and more explicit. So it is better but in a very subjective way. In theory, mypy can catch the absence of Optional accurately in both cases.